app/Http/Requests/Portfolio/UpdatePortfolioRequest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Requests\Portfolio;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdatePortfolioRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'name' => ['required', 'string'],
+            'defualt_currency' => ['required', 'string'],
```

Comment:
I was just passing by and saw this typo `default_currency` ?  😄

Reply:
Got it fixed, thanks for pointing that out!